packages/store-ui/src/atoms/Price/Price.tsx

```diff
@@ -0,0 +1,49 @@
+import type { HTMLAttributes, ElementType, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+export type PriceFormatter = (price: number, variant: PriceVariant) => ReactNode
+
+export type PriceVariant =
+  | 'default'
```

Comment:
is it "selling" price the "default" in here?

Reply:
You're right! I was trying to find the name and just couldn't.